4_arm.ipynb

```diff
@@ -157,26 +282,42 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>課題</b>\n",
     "    <p>「モノを拾う」アームの動きを作ってみましょう。</p>\n",
     "</div> "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Create an arm motion to \"pick the object\".</p>\n",
```

Comment:
"    <p>Create an arm motion to \"pick up an object\".</p>\n",

拾う = pick up
And because we have not specified a particular object yet, we replace "the" with "an".